src/Containers/packaging/build/Microsoft.NET.Build.Containers.buildschema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": {
+    "EnableSdkContainerSupport": {
+      "type": "bool",
+      "defaultValue": "false",
+      "description": "Enables the ability to package a project in a container. This is enabled by default if the project is an ASP.NET project or a Worker Project, or if the project includes a references to the Microsoft.NET.Build.Containers package."
+    },
+    "ContainerBaseImage": {
+      "type": "string",
+      "defaultValue": "mcr.microsoft.com/dotnet/runtime:8.0",
```

Comment:
8.0 intentional since this is main?

Reply:
Oh that file shouldn't even be checked in, it's not relevant to this template change at all. Let me revert it.